plugins/experimental/parent_select/consistenthash_config.cc

```diff
@@ -108,8 +108,8 @@ createStrategiesFromFile(const char *file)
 
     // std::map<std::string, TSNextHopSelectionStrategy*, std::less<>>
     strategies_map strategiesMap;
-    for (unsigned int i = 0; i < strategies.size(); ++i) {
-      YAML::Node strategy = strategies[i];
+    for (auto &&strategie : strategies) {
+      YAML::Node strategy = strategie;
```

Comment:
There were two `strategie` variables in the diff. The second is address, but this remains.

Look out though: renaming this to strategy will also clash with the inner `strategy` variable. We can potentially change `strategie` to `strategy`, and the inner strategy to `strategy_copy`?

Thanks for working on this.